frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,95 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-user-preferences"
+              And user is at Add page
```

Comment:
IMO, we can remove this steps, because user settings are available on MastHead. WDYT?